src/context/viewer-context.tsx

```diff
@@ -82,22 +82,23 @@ function useAuthedViewer() {
       }
     }
 
-    if (authToken) {
-      const sixtyDaysInSeconds = JSON.stringify(60 * 24 * 60 * 60)
-      auth
-        .handleCookieBasedAccessTokenAuthentication(
-          authToken,
-          sixtyDaysInSeconds,
-        )
+    const loadViewerFromToken = async () => {
+      await auth
+        .handleCookieBasedAccessTokenAuthentication(authToken)
         .then((refreshedViewer: any) => {
           setViewer(refreshedViewer)
-          console.log(`refresh user from cookie auth token`)
           setLoading(() => false)
         })
         .catch((error) => {
           // if it is a 403, clear out the token and then call auth.monitor?
+          // auth does a logout on any error currently -jh
```

Comment:
I think `auth` does this under the hood so we should be good?

Reply:
yep, I removed the catch and flattened this out https://github.com/eggheadio/egghead-next/pull/183/commits/a69c120641cad9feed568a4dfd94066483924326